tests/providers/jenkins/operators/test_jenkins_job_trigger.py

```diff
@@ -20,15 +20,21 @@
 
 import jenkins
 import mock
+from parameterized import parameterized
 
 from airflow.exceptions import AirflowException
 from airflow.providers.jenkins.hooks.jenkins import JenkinsHook
 from airflow.providers.jenkins.operators.jenkins_job_trigger import JenkinsJobTriggerOperator
 
 
+@unittest.skipIf(mock is None, 'mock package not present')
```

Comment:
Coming to think of it, I think this line doesn't make sense. Wouldn't we need to add this up in the imports?

```python
try:
    import mock
except ImportError:
    mock = None
```

Reply:
Let's just do `from unittest import mock` and remove the skipIf